src/derivative_operators/convolutions.jl

```diff
@@ -21,11 +21,11 @@ function convolve_interior!(x_temp::AbstractVector{T}, x::AbstractVector{T}, A::
         for idx in 1:A.stencil_length
             xtempi += cur_coeff * cur_stencil[idx] * x[i - mid + idx]
         end
-        x_temp[i] = xtempi
+        x_temp[i] = xtempi + !:($overwrite)*x_temp[i]
```

Comment:
that's a weird way to write `!overwrite * x_temp[i]`

Reply:
Yes... I should change that. I think I was originally trying to do something more convoluted with metaprogramming but forgot to get rid of the inference.